app.py

```diff
@@ -156,7 +158,7 @@ def main():
             schedule=json.dumps(get_schedule(session["username"])),
             days=json.dumps(DAYS),
             components="static/components.html",
-            lunches="[]",
+            lunches=get_lunch_for_date(datetime.date.today()),
```

Comment:
This is the line that's causing the problem. I think what you want to do here is define a new variable called `LUNCHES` and initialize it in `init_app()`, like `SCHEDULES` and `DAYS`. Then we can do special initialization of `LUNCHES` when running `main_test.py`.

Let me know if this makes sense, otherwise I can make an example PR.